src/test/e2e/api/openapi/__snapshots__/openapi.e2e.test.ts.snap

```diff
@@ -195,6 +195,11 @@ Object {
           "archived": Object {
             "type": "boolean",
           },
+          "archivedAt": Object {
```

Comment:
Should we remove `archived` from the response? Or do we want to avoid breaking the API?

Reply:
That is the reason I let archived in. Didn't want to break the api